packages/dx-react-grid-material-ui/src/templates/table.jsx

```diff
@@ -12,7 +12,6 @@ const styles = {
     position: 'sticky',
     top: 0,
     zIndex: 1,
-    background: 'white',
```

Comment:
Maybe we should use color from the `theme` object? `theme.palette.background.paper` for example.

Reply:
Sure that would be better.